crates/bevy_text/src/text.rs

```diff
@@ -284,6 +284,11 @@ pub struct TextFont {
     /// A new font atlas is generated for every combination of font handle and scaled font size
     /// which can have a strong performance impact.
     pub font_size: f32,
+    /// The vertical height of a line of text in pixels, from the top of one line to the top of the
+    /// next.
+    ///
+    /// If not set, the line height will be 1.2 * `font_size`.
+    pub line_height: Option<f32>,
```

Comment:
Is there a point to making this optional, it could just be an `f32` with a default value of 1.2 instead?

Reply:
It's measured in pixels, not multiples of the font size (though that might be a better interface). Working with `Options` makes it simpler to implement too IMO, but changing it wouldn't be a huge deal

Reply:
Oh I see it's not a scalar. I think then it would better if there was a `LineHeight` enum maybe with `Px` and `Scalar` variants or something.

Reply:
I like that! Best of both worlds

Reply:
in that case, should it be a field of `TextFont`, or its own component?

Reply:
Otherwise it looks good, the UI text debug examples are still working with different line heights, don't see any problems.

Reply:
> in that case, should it be a field of `TextFont`, or its own component? Being a separate component would make it so it's not a breaking change at all, but I'm not sure yet how to do that

The current direction atm with bevy seems to be less granular APIs so a field on `TextFont` is preferable probably. A field is more discoverable than a distinct component as well.